haddock-api/src/Haddock/Interface.hs

```diff
@@ -165,6 +167,7 @@ createIfaces verbosity flags instIfaceMap mods = do
 processModule :: Verbosity -> ModSummary -> [Flag] -> IfaceMap -> InstIfaceMap -> Ghc (Maybe Interface)
 processModule verbosity modsum flags modMap instIfaceMap = do
   out verbosity verbose $ "Checking module " ++ moduleString (ms_mod modsum) ++ "..."
+  liftIO $ hSetEncoding stderr utf8
```

Comment:
I finally found a workaround:

```haskell
import GHC.IO.Encoding.CodePage
import GHC.IO.Encoding.Failure
import System.IO

hSetEncoding stderr $ mkLocaleEncoding TransliterateCodingFailure
```

I'm trying to update the patch and rebase, but I have a difficulty testing with GHC 8.2.
The latest master is now for GHC 8.2, but I failed to install GHC 8.2 on my Windows... 😞